torch/onnx/_globals.py

```diff
@@ -28,7 +27,9 @@ def __init__(self):
         self._in_onnx_export: bool = False
         # Whether the user's model is training during export
         self.export_training: bool = False
-        self.operator_export_type: Optional[_C_onnx.OperatorExportTypes] = None
+        self.operator_export_type: _C_onnx.OperatorExportTypes = (
```

Comment:
I am planning on adding a new type (https://github.com/pytorch/pytorch/issues/84884). Any suggestion for that?

Reply:
Addressed offline